docs/platform-integration/snippets/shared_1/MainPage.xaml.cs

```diff
@@ -53,5 +53,10 @@ private void Media_Clicked(object sender, EventArgs e)
     {
         Navigation.PushAsync(new MediaPage());
     }
+
+    private void Storage_Clicked(object sender, EventArgs e)
```

Comment:
Shouldn't this be `private async void` and `Navigation.PushAsync` should be `await Navigation.PushAsync`?